elfpy/markets/hyperdrive/hyperdrive_market.py

```diff
@@ -289,6 +292,25 @@ def spot_price(self) -> float:
             time_remaining=self.position_duration,
         )
 
+    def collect_gov_fee(self, governance_wallet: wallet.Wallet):
```

Comment:
From what I can tell, this function is only used in the tests, so it should live there. This keeps code closer to where it is used, and also in general the market should not be directly updating an agent's wallet. Alternatively, in the test you should be able to get the fee information via the agent_delta Wallet and then update it at the test level. Then you can delete `collect_gov_fee` all together. That's my preference.

Reply:
done in 95eb80fe112de5f6a24b9cb3aebf1f720f95e83b